doc/source/user_guide/numpy_images.rst

```diff
@@ -147,26 +152,35 @@ denote standard Cartesian coordinates, where ``x`` is the horizontal coordinate,
 ``y`` - the vertical one, and the origin is at the bottom left
 (Matplotlib axes, for example, use this convention).
 
-In the case of multichannel images, the last dimension is used for color channels
-and is denoted by ``channel`` or ``ch``.
+In the case of multichannel images, any dimension (array axis) can be used for
+color channels, and is denoted by ``channel`` or ``ch``. Prior to scikit-image
+0.19, this channels dimension was always last, but in the current release the
```

Comment:
```suggestion
0.19, this channel dimension was always last, but in the current release the
```
Not wrong, but I realize the following line reads 'the channel dimension' (and it saves one character). Sorry, I'm the nitpickiest!! :innocent: